pkg/apis/operator/v1alpha1/catapult_types.go

```diff
@@ -36,6 +36,8 @@ type CatapultSpec struct {
 	// ServiceCluster: Webhook will call webhooks of the CRD in the ServiceCluster with dry-run flag.
 	// +kubebuilder:default:=None
 	WebhookStrategy corev1alpha1.WebhookStrategyType `json:"webhookStrategy,omitempty"`
+	// Paused tell controller to pause reconciliation process and assume that Catapult is ready
+	Paused bool `json:"paused,omitempty"`
```

Comment:
How about we use `string` type `PausedCondition`: (`True`, `False`) for this? I think boolean variable is not the preferable way in Kubernetes since it is hard to extand.

Reply:
yeah, good point, always forget about that

Reply:
done